charts/hami/templates/device-plugin/daemonsetnvidia.yaml

```diff
@@ -63,6 +63,10 @@ spec:
               value: all
             - name: HOOK_PATH
               value: {{ .Values.global.gpuHookPath }}
+            {{- if typeIs "bool" .Values.devicePlugin.compatWithCPUManager }}
+            - name: PASS_DEVICE_SPECS
```

Comment:
Because hami no implement read this env PASS_DEVICE_SPECS logic, so even if add it after code is invalid.

Reply:
Yes, I also had this issue at the beginning, including the fact that the nvidia-k8s-device plugin only recognized PASS_DEVICE_SPECS but did not use it. However, adding PASS_DEVICE_SPECS env is effective, perhaps due to compatibility with nvidia-k8s-device plugin causing it to take effect?